cli/aws_ddk/sh.py

```diff
@@ -28,18 +28,19 @@ def _clean_up_stdout_line(line: bytes) -> str:
     return line_str[:-1] if line_str.endswith("\n") else line_str
 
 
-def _run_iterating(cmd: str, cwd: Optional[str] = None) -> Iterable[str]:
-    p = subprocess.Popen(shlex.split(cmd), cwd=cwd, stdout=subprocess.PIPE, stderr=subprocess.STDOUT)
+def _run_iterating(cmd: str, cwd: Optional[str] = None, **kwargs: Any) -> Iterable[str]:
```

Comment:
maybe something more descriptive than `kwargs` like `subprocess_args` would be better

Reply:
It's all right, it's usual convention for keyword arguments. `subrocess_kwargs` also would do